src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -12,11 +15,30 @@ public CountryDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Country add(Country country) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(country);
+            transaction.commit();
+            return country;
+        } catch (RuntimeException e) {
```

Comment:
The same, fix everywhere
```suggestion
        } catch (Exception e) {
```